carl_interactive_manipulation/CMakeLists.txt

```diff
@@ -27,10 +41,16 @@ include_directories(
 
 ## Declare a cpp executable
 add_executable(carl_interactive_manipulation src/carl_interactive_manipulation.cpp)
+add_executable(carl_parking src/create_parking_spots.cpp)
 
 ## Specify libraries to link a library or executable target against
-target_link_libraries(carl_interactive_manipulation
-  ${catkin_LIBRARIES}
+target_link_libraries(
+	carl_interactive_manipulation
+	${catkin_LIBRARIES}
+)
+target_link_libraries(
+	carl_parking
```

Comment:
match node name with CPP file name. either rename the cpp file to `carl_parking.cpp` or rename executable to `create_parking_spots`